Article/Article.md

```diff
@@ -22,110 +31,80 @@ abstract: |
+$p_i = \beta_1^c female_i + \beta_2^c age_i + \beta_3^c age^2_i + \beta_4^c subjective \ math \ skills_i + \epsilon_i$
 
-The power and limitation of WEF Global Gender Gap index remains mostly undiscussed in the academic literature, with a few exceptions [@Nicaragua; @10.1080/13545701.2010.530607]. The index is thought to be the least dependent to the economic development of a country, since it measures the gap between male and female access to resources and opportunities [@WEF_report].
+This results in 6 models -- one for each preference measure, $p_i$ -- having intercept and 4 coefficients, each of them being related to the variable in the formula above. The coefficient for the dummy variable "female", $\beta_1^c$, is used as a measure of the country-level gender difference. Therefore, in total, we have 6 coefficients that represent the preference differences related to the gender for 76 countries. To summarise the gender differences among the six economic preferences, a principal component analysis (PCA) is performed on the gender coefficients. The PCA is a dimensionality-reduction technique which allows to “reshape” the 6 coefficients into other mixed components that maximizes the variance. The first component of the PCA has then been used as a summary index of average gender differences in preferences.
 
-<!--
-Here the part about suffrage:
-Worthy to mention also the question about the connection between the suffrage and race, that for many countries is strongly connected [@Yang; @NBERw20864]. has always a positive effect and its magnitude is not always proportional to the time since women suffrage was established, and the magnitude of this effect is proportional to the time since women suffrage.  Even after the right to vote was granted, discriminating laws may still be in presence, and the alignment of law together with executive branch of the government and elimination of discrimination takes more time -- for example, despite to the right to vote, the right to work can be suppressed for several decades. 
--->
+We managed to obtain gender economic preferences to economic preferences for all countires, as well as conduct a pure replication of the results of the original article (see Appendix. 1 for more details). In addition to this, as the diagnostic of the lienar regressions revealed a presence of non-normality, we run an analysis based on robust linear regression instead of ordinary linear regression to insure the the following effects don't lead to any biases in the results.
```

Comment:
We managed to replicate the analysis and to obtain the gender differences in the economic preferences at the country level (for details, see Table 1 in the Appendix).